voxblox/include/voxblox/simulation/simulation_world_inl.h

```diff
@@ -37,8 +37,13 @@ void SimulationWorld::generateSdfFromWorld(FloatingPoint max_dist,
         layer->allocateBlockPtrByIndex(block_index);
     for (size_t i = 0; i < block->num_voxels(); ++i) {
       VoxelType& voxel = block->getVoxelByLinearIndex(i);
-      Point coords =
-          block->computeCoordinatesFromLinearIndex(i);
+      Point coords = block->computeCoordinatesFromLinearIndex(i);
+      // Check that it's in bounds, otherwise skip it.
```

Comment:
Before it was just by-block, and it made the map look sloppy (i.e., much longer on one side depending on where the block boundaries are). I think for visualization purposes it's better to just stick to the actual requested boundaries.